### PR TITLE
Move to using root_base

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
Closes #9 and this PR should be fixed by https://github.com/conda-forge/root-feedstock/pull/68

**Comment by chrisburr** [comment]
This PR is needed due to changes I've made in the `root` recipe which make it possible to install without including all of it's dependencies (hence it should be faster and use less disk space). This also means that the `root` command line program is not available after installing only `ostap`. If you have any issues with this new set up let me know.

@TatianaOvsiannikova It looks `configparser` should be added as a Python 2.7 dependency in ostap's `setup.py`. Can you deal with it? Also feel free to push any other changes you have to this branch before merging (e.g. updating the version number).

**Comment by TatianaOvsiannikova** [comment]
Thanks a lot! Yes, the configparser and future are required only for python 2.7.  I will add other new dependencies as soon as possible and merge your request. In the following version of ostap we are going to add functionality which depended on the hadd. It's not included in root_base, right?

**Comment by chrisburr** [comment]
Then you also need `root-binaries` in the `run` dependencies.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,45 +22,53 @@ requirements:
...
   run:
-    - python 
-    - root 
+    - python
     - future
     - ipython
-    - gsl
+    - configparser  # [py2k]
+    - scipy
+    - numpy
```
Comment (chrisburr) [inline recipe/meta.yaml]: Do you use the numpy C API? Or is numpy only used from Python?

Comment (TatianaOvsiannikova) [reply recipe/meta.yaml]: No, it's used from Python. Should we not specify it as a separate dependence?

Comment (chrisburr) [reply recipe/meta.yaml]: No in that case this is perfect. If you were using the C API you'd need [to do it this way](https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy) to account for the ABI stability.

**Pull request merged**